docs/learn/learn-agile-coretime.md

```diff
@@ -0,0 +1,132 @@
+
+### On-demand Coretime
+
+Previously known as instantaneous coretime, the on-demand coretime refers to coretime acquired 
+through bidding in near real-time for the validation of a single parachain block on one of the 
+cores reserved specifically for on-demand orders.
+
+### On-demand Coretime Pool
+
+Set of cores that are available on-demand. Cores reserved through bulk coretime could also be
+made available in the on-demand coretime pool, in parts or in entirety.
+
+### Coretime Chain
+
+A system parachain that is responsible for the sale, manipulation of bulk coretime and 
+eventually the purchase of on-demand coretime credits. It is also responsible for 
+scheduling the respective cores on the relay chain to the parachains.
+
+### Timeslice
+
+A timeslice represents the granularity the Coretime Chain schedules cores on the Relay Chain. 
+The Coretime Chain announces a schedule for each core, comprising the next 80 relay 
+chain blocks in advance. The region length of 28 days corresponds to 5,040 timeslices.
+
+### Task
+
+The term "Task" captures the abstraction of utility of the core. Tasks on the cores
+are not just limited to securing parachains.
+
+## Agile Coretime Implementation
+
+Coretime is managed through the [broker pallet](https://github.com/paritytech/polkadot-sdk/blob/eefd5fe4499515da66d088505093f75b5aa22550/substrate/frame/broker/src/lib.rs) 
+which is deployed on the Coretime Chain. In theory, the Polkadot Relay Chain can support around a 
+hundred cores seamlessly and can support over a few hundred cores through optimizations. Preliminary tests ran successfully with 80 cores with 12-second block times.
+
+### Coretime Sales
+
+Sales on the Coretime Chain are of `timeslice` length. These sales are divided into two main periods: the Renewal period and the Sale period.
+
+![coretime-sale-timeline](../assets/coretime/Coretime-Sale-Timeline.png)
+
+- Renewal periods are of `interlude_length` blocks long, and it’s where bulk coretime can be renewed.
+- Sale periods are as well divided into two periods: a Price Discovery period and a Fixed Price period. After the Renewal period, a new `start_price` will be set by the Coretime Chain and a dutch auction of `leading_length` blocks will start, putting downward pressure on price to find the right equilibrium. This process will set the `regular_price` which will be the one offered at the Fixed Price period.
+
+This discussion on initial coretime pricing can be viewed [here](https://forum.polkadot.network/t/initial-coretime-pricing/5187)
+
+### Splitting and Interlacing
+
+Splitting and interlacing are actions that can be performed within a region. Performing either of 
+these actions makes you lose the right to a price-capped renewal.
+
+- **Splitting**: the action of dividing a region into several regions with different start and end 
+timeslices.
+- **Interlacing:** the action of dividing a region into the execution of different tasks for each 
```

Comment:
Can we give a practical real example here about splitting and interlacing? It is a bit hard to follow

Reply:
Yeah, this will be part of another PR that provides more info through guides and tutorials.